.azure-pipelines/azure-pipelines-build.yml

```diff
@@ -110,6 +110,25 @@ jobs:
     buildSteps:
       - template: template-skipvstest.yml
       - bash: |
+          (
```

Comment:
Can we add the step in a template, and the release branch use the same template?